src/server/cookie.ts

```diff
@@ -0,0 +1,51 @@
+import cookieParser from "cookie-parser";
+import { Application } from "express";
+import { DAILY_API_KEY } from "./env";
+
+type Cookies = { [key in string]?: string };
+
+// setupMiddleware() has our application use the cookie parser
+// with the Daily API key as our cookie signing secret.
+export function setupCookieParser(app: Application) {
+  app.use(cookieParser(DAILY_API_KEY));
```

Comment:
Right now for simplicity we use the Daily API key to sign our cookies on the server-side.

Reply:
Not a big deal, but I might sign these with a different secret, mainly so that you could rotate this secret independent of rotating the Daily API token.

Reply:
Mm yeah good point. I might have it take another env variable with the signing secret, and auto-generate one per server run if a persistent env var is not provided.